bin/extract-translations.sh

```diff
@@ -11,6 +11,6 @@ echo '# Extracting translations from vendor/ezsystems/hybrid-platform-ui';
 cd vendor/ezsystems/hybrid-platform-ui;
 
 echo '# Clean file references';
-sed -i .bak "s|>.*/hybrid-platform-ui/|>|g" ./src/bundle/Resources/translations/*.xlf
+sed -i "s|>.*/hybrid-platform-ui/|>|g" ./src/bundle/Resources/translations/*.xlf
```

Comment:
Well, without this, it fails on mac... :)

There's probably a better solution, but I don't remember it.